### PR TITLE
Added Get Quotes Command

### LINKED ISSUE #57: GetQuotes Command

### PR DESCRIPTION
**Pull request by zaydtahir**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: Closes #57 

## What is the new behavior?

- Added get quote command

### CODE REVIEW

**Comment by zaydtahir** [comment]
I anticipate merge conflicts from #70 so make sure that gets merged first.

File: src/commands/Quotes/addQuote.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {addQuote} from '../../database/models/QuoteModel';
 
 const {Command} = require('@sapphire/framework');
 
-const EMBED_FIELD_MAX_CHAR_LENGTH = 1024;
+const EMBED_FIELD_MAX_CHAR_LENGTH = 1022;
```
Comment (sahil-kale) [inline src/commands/Quotes/addQuote.ts:11]: Why the lowering?

File: src/commands/Quotes/getQuotes.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import {PaginatedMessage} from '@sapphire/discord.js-utilities';
+import {getQuotesFromAuthor} from '../../database/models/QuoteModel';
+import {BediEmbed} from '../../lib/BediEmbed';
+import colors from '../../utils/colorUtil';
+import {surroundStringWithBackTick} from '../../utils/discordUtil';
+import {getSettings} from '../../database/models/SettingsModel';
+
+const {Command} = require('@sapphire/framework');
+
+const MAX_QUOTES_PER_PAGE = 5;
```
Comment (sahil-kale) [inline src/commands/Quotes/getQuotes.ts:12]: Can we up this number to 10?

File: src/commands/Quotes/getQuotes.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    let templateDescription;
+
+    if (typeof quoteAuthor.value === 'string') {
+      templateDescription = `Quotes by ${surroundStringWithBackTick(quoteAuthor.value)}`;
+    } else {
+      templateDescription = `Quotes by ${quoteAuthor.value}`;
+    }
+
+    const paginatedMessage = new PaginatedMessage();
```
Comment (sahil-kale) [inline src/commands/Quotes/getQuotes.ts:66]: Comment on this line please

File: src/database/models/QuoteModel.ts [hunk 4]
```diff
@@ -33,4 +33,13 @@ export const removeQuote = async (guildId: string, quote: string, author: string
...
 
+export const getQuotesFromAuthor = async (guildId: string, author: string) => {
+  return quoteModel.find({
+    guildId: guildId,
+    author: {
+      $regex: new RegExp(`^${author}$`, 'i'),
+    },
+  });
+};
+
```
Comment (sahil-kale) [inline src/database/models/QuoteModel.ts:44]: Can this function be tested?

**Review by sahil-kale — CHANGES_REQUESTED** [review]

File: src/commands/Quotes/addQuote.ts (hunk 1 above)
Comment (zaydtahir) [reply src/commands/Quotes/addQuote.ts:11]: I display the command using back ticks to make it look nicer, so I'm accounting for those two characters.

**Review by sahil-kale — APPROVED** [review]

**Pull request merged**